src/platform/forms-system/src/js/components/ReviewCardField.jsx

```diff
@@ -36,7 +37,7 @@ export default class ReviewCardField extends React.Component {
 
     // Throw an error if there’s no viewComponent (should be React component)
     if (
-      typeof get('ui:options.viewComponent', this.props.uiSchema) !== 'function'
+      !isReactComponent(get('ui:options.viewComponent', this.props.uiSchema))
```

Comment:
Basically any type check for function needs to be evaluated, because components using `Provider` are no longer functions.